### PR TITLE
Feature/wmt 678/add seed directory

### PR DESCRIPTION
**Pull request by andrew-js-wright**
Database object which don't require the write once functionality provided by
migrations should be handled using seeds. This means that each files is ran on
each deployment.

This gives us the flexibility of being able to change database views without
needing to create a new file.

These are numbered sequentially and can be run using the following command:

`yarn seed-views`

### CODE REVIEW

File: cleanup.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const knex = require('knex')(config)
 const glob = require('glob')
 const Promise = require('bluebird').Promise
 
-var seedFileNames = glob.sync('./data/[0...9]*.js')
+var seedFileNames = glob.sync('./seed/data/[0...9]*.js')
```
Comment (caoimheb) [inline cleanup.js]: Maybe a name that makes it clear this is just the data seed files?

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "integration-test": "mocha --recursive -u exports test/integration/ --timeout 10000",
     "test-all": "yarn test && yarn test-integration",
     "test": "yarn unit-test && yarn integration-test",
-    "migrate-app": "knex migrate:latest --env app",
+    "create-views": "knex seed:run --env views",
```
Comment (caoimheb) [inline package.json]: Description on the PR says `yarn seed-views`. Preference?

File: seed/data/01000_workload_points_calculations.js [hunk 3]
```diff
@@ -24,11 +24,26 @@ exports.seed = function (knex, Promise) {
...
+          effective_from: effectiveFromDate,
+          effective_to: effectiveFromDate.getDate() + 30,
+          total_points: Math.floor(Math.random() * 25) + 180,
+          available_points: 190,
+          paroms_points: 50,
+          sdr_conversion_points: 50,
+          sdr_points: 50,
+          nominal_target: 0,
+          reduction_hours: Math.floor(Math.random() * 6) + 1
+        })
```
Comment (caoimheb) [inline seed/data/01000_workload_points_calculations.js]: 👍🏻

**Comment by caoimheb** [comment]
Can we order the capacity views as team, ldu, region, national?

File: package.json (hunk 2 above)
Comment (andrew-js-wright) [inline package.json]: whoops, nice spot. I think I'll stick with `create-views`.

**Pull request merged**